packages/wallets/src/components/Base/WalletAlertMessage/WalletAlertMessage.tsx

```diff
@@ -34,19 +34,19 @@ const WalletAlertMessage: React.FC<TProps> = ({ children, message, type }) => {
     const color = typeMapper[type].color;
 
     return (
-        <div className={styles['wallets-alert-message']} data-testid='dt_wallet-alert-message'>
-            <div className={styles['wallets-alert-message__icon-container']}>
-                <div className={styles['wallets-alert-message__icon-container__line']} />
-                <div className={styles['wallets-alert-message__icon-container__icon']}>
+        <div className={'wallets-alert-message'} data-testid='dt_wallet-alert-message'>
```

Comment:
non-blocking: remove bracket `{}` because its just literal string for className

Reply:
@farhan-nurzi-deriv sure sure, forgot to remove them during refactoring to scss